src/benchmark/file_io_read_micro_benchmark.cpp

```diff
@@ -21,10 +22,10 @@ class FileIOMicroReadBenchmarkFixture : public MicroBenchmarkBasicFixture {
 
   void SetUp(::benchmark::State& state) override {
     // TODO(everybody): Make setup/teardown global per file size to improve benchmark speed
-    ssize_t BUFFER_SIZE_MB = state.range(0);
+    ssize_t BUFFER_SIZE = allign_to_pagesize(state.range(0));
```

Comment:
`allign_to_pagesize` seems like a good function! Should we use it everywhere? Why have you used it?

Reply:
Because umap() requires the mapped range be a multiple of the pagesize.
I'd like to use it everywhere.

Reply:
Then we should do it like that :)

Reply:
Typo in 'allign_to_pagesize' -> 'align_to_pagesize'